tests/parametric/test_config_consistency.py

```diff
@@ -610,3 +612,49 @@ def test_config_stable(self, library_env, test_agent, test_library):
             assert (
                 config["dd_service"] == "my-service"
             ), f"Service name is '{config["dd_service"]}' instead of 'my-service'"
+
+    @missing_feature(
+        context.library in ["ruby", "cpp", "dotnet", "golang", "nodejs", "php", "python"],
+        reason="UST stable config is phase 2",
+    )
+    @pytest.mark.parametrize(
+        "library_extra_command_arguments",
+        [
+            ["-Darg1=value"]
```

Comment:
The issue here is that `missing_feature` stills execute the test, and is silent if ti fails. Though, it does not allow the container to fail at startup, which happens on some tracers here.

There is two solutions : 

* the bad one , add `force_skip=True` argument  at then end of  `missing_feature`
* the good one : replace `["-Darg1=value"]` by something supported by each language